mmv1/third_party/terraform/tests/resource_compute_instance_template_test.go.erb

```diff
@@ -3155,8 +3155,12 @@ data "google_compute_image" "debian" {
   project = "debian-cloud"
 }
 
+resource "random_id" "id" {
+	  byte_length = 8
+}
+
 resource "google_compute_disk" "persistent" {
-  name  = "debian-disk"
+  name  = "debian-disk-${random_id.id.hex}"
```

Comment:
Instead of using the `random_id` provider, could we instead use the `suffix` variable that is passed to this function? It uses `randString()`, which is actually not random for the VCR tests that run for PRs, and allows REPLAY mode to work.

Also, we should probably add a `tf-test-` prefix, so that if the test fails, the sweeper cleans up the leftover disk.

Reply:
Good Idea, ill add this